lib/cdo/poste.rb

```diff
@@ -23,7 +23,10 @@ def self.decrypt(encrypted)
   end
 
   def self.decrypt_id(encrypted)
-    return decrypt(encrypted).to_i
+    decrypt(encrypted).to_i
+  rescue OpenSSL::Cipher::CipherError, ArgumentError => e
+    CDO.log.warn "Unable to decrypt poste id: #{encrypted}. Error: #{e.message}"
```

Comment:
That is possible. I'm really not sure why we've been getting bad urls, and thus far in honeybadger I haven't been able to see the actual url or params. I'll leave in the warning so we get a record of these. Given that we won't be seeing honeybadger errors, we likely will ignore it.

Adding documentation re: return value